### DISCUSSION TITLE
Choosing color of the nodes and edges

### DISCUSSION THREAD

**Discussion by mth-farias** [talmolab/sleap#999] [Help!] [answered]
Can I define the color of each node and edge? How?

I cannot find a similar discussion

Thanks a lot
Matheus

**Comment by roomrys** [comment, marked as answer]
Hi @mthfarias21,

SLEAP currently allows the users to select from a few pre-defined color pallets (View > Color Palette). SLEAP also allows users to specify how to color instances (View > Apply Distinct Colors To).

Let me know if this helps!

Thanks,
Liezl

EDIT: See @mthfarias21's answer below for defining the colors in the color palette: 
> Just found how to do it on the website
> 
> 
> 
> "Color palettes can be customized by modifying the colors.yaml file that SLEAP creates inside the .sleap directory in your home directory. You can add your own palette or modify those already present in the file. Each color in a palette is listed on it’s own line as r,g,b values (between 0 and 255)." (https://sleap.ai/guides/gui.

**Reply by mth-farias** [reply]
Hi @roomrys !

Thank you very much for the reply! This helps a lot!

It would be nice if the user gets to choose the color of each node and edge, it would make it easier to visualy indetify left/right, anterior/posterior limbs. This question came to me during the prediction assisted labelling because it would make it easier if for instance I could label the left side green and right side red, with the anterior/posterior axis being instensity of those colors.

The existing feature already helps al ot, I was just wondering if it would be possible

Cheers,
Matheus

**Reply by mth-farias** [reply]
Just found how to do it on the website

"Color palettes can be customized by modifying the colors.yaml file that SLEAP creates inside the .sleap directory in your home directory. You can add your own palette or modify those already present in the file. Each color in a palette is listed on it’s own line as r,g,b values (between 0 and 255)." (https://sleap.ai/guides/gui.html)

This is great! Thanks

**Reply by roomrys** [reply]
@mthfarias21  Oof, I overlooked that - my bad! Thanks for sharing the info!

Also, shameless plug: if you are looking for a [rgb color picker](https://roomrys.github.io/rgb-axis/)
